src/documentation/pages/components/date-picker/examples/_base-date-picker.vue

```diff
@@ -0,0 +1,52 @@
+<template lang="pug">
+  <ComponentExample title="Base" id="example__base_date_picker" :tabs="exampleTabs">
```

Comment:
Here you have to continue doing the same as before, just apply the changes in the ids of the examples.

```suggestion
  <ComponentExample title="Base" id="base-date-picker" :tabs="exampleTabs">
```
Check this in each example.